pkg/diff/merge_test.go

```diff
@@ -31,12 +30,12 @@ func (s *testMergerSuite) TestMerge(c *C) {
 	c.Assert(err, IsNil)
 
 	_, orderKeyCols := dbutil.SelectUniqueOrderKey(tableInfo)
-	ids := []int{3, 2, 2, 4, 1}
-	names := []string{"d", "b", "c", "b", "a"}
-	ages := []int{1, 2, 3, 4, 5}
+	ids := []string{"3", "2", "2", "4", "1", "NULL"}
+	names := []string{"d", "NULL", "c", "b", "a", "e"}
```

Comment:
we can add more test cases to increase code path coverage later (easy to find out which code is not covered after added some coverage integration tool).

Reply:
ok